syntax = "proto2";

package botCode;


message PacCommand {

  message Command {
    required Direction direction = 1;
    required int32 forwards_distance = 2;
  }
  
  required Command command_1 = 1;
  required Command command_2 = 2;
  required Command command_3 = 3;
  required Command command_4 = 4;
  required Command command_5 = 5;

  enum Direction {
    LEFT = 0;
    RIGHT = 1;
    FORWARDS = 2;
  }
}